### DISCUSSION TITLE
Concern about blade displacement

### DISCUSSION THREAD

**Discussion by peterfakhry** [OpenFAST/openfast#1136] [Q&A]
Hello there, thanks a lot for the considerable efforts in producing OpenFAST. I am still in a phase of validating my numerical model that solves structural blade responses (in plan and out of plan) using OpenFAST. I have a concern about the shape of vibration generated using OpenFAST. Why do we have this 'wavy' shape like as shown below (i don't mean the harmonic overall response but rather the local wavy behavior). 
![image](https://user-images.githubusercontent.com/40067011/169562817-ba914444-aaa7-48a2-9a1d-36d8616f1e9e.png)

**Comment by jjonkman** [comment]
Dear @peterfakhry,

I'm guessing that the response is driven by excitation and or modes that involve multiple frequencies.  One component has a frequency of about 0.25 Hz and the other as a frequency of about 1 Hz.

Best regards,

**Reply by peterfakhry** [reply]
Thanks a lot for you reply professor. i am simulating against a steady wind with speed = 12m/sec and still getting this wavy behavior
![Screenshot 2022-05-21 143727](https://user-images.githubusercontent.com/40067011/169653536-db3f5e48-2ffb-40fe-9d9e-f55c5fd3f8e4.png)
Another thing is it possible to simulate OpenFAST with a constant rotor speed ( I already turned of the servo controller but also want a constant rotor speed?)
thanks

**Reply by jjonkman** [reply]
Dear @peterfakhry,

To simulate OpenFAST at a fixed rotor speed regardless of the loads applied, disable the generator degree of freedom in ElastoDyn (`GenDOF` = False).

Best regards,
